### PR TITLE
Complete any class name inside attribute

### PR DESCRIPTION
**Pull request by przepompownia**
It only allows what the title says. Classes that are not attributes are also suggested with this change.

### CODE REVIEW

**Comment by przepompownia** [comment]
@dantleech what do you think about merging it before working on #1999?

File: lib/Completion/Bridge/TolerantParser/CompletionContext.php [hunk 1]
```diff
@@ -51,6 +52,7 @@ public static function expression(?Node $node): bool
             $parent instanceof Expression ||
             $parent instanceof StatementNode ||
             $parent instanceof ConstElement ||
+            $parent instanceof Attribute ||
```
Comment (dantleech) [inline lib/Completion/Bridge/TolerantParser/CompletionContext.php:55]: it's not valid to have an expression in an attribute though... !?

Comment (przepompownia) [reply lib/Completion/Bridge/TolerantParser/CompletionContext.php:55]: Maybe it's a hack.  Would that mean that it should be checked separately, especially in some new Completor mentioned in #1999?

Comment (dantleech) [reply lib/Completion/Bridge/TolerantParser/CompletionContext.php:55]: yes, probably there should be a `attributeClass(Node $node)` method here - but equally can be done in the new `Completor`

**Comment by przepompownia** [comment]
Let's focus on #2001 instead.

**Pull request closed without merging**